patroni/dcs/consul.py

```diff
@@ -266,20 +278,20 @@ def reload_config(self, config):
         self._previous_loop_register_service = self._register_service
         self._register_service = should_register_service
 
-    def set_ttl(self, ttl):
+    def set_ttl(self, ttl: int) -> Optional[bool]:
```

Comment:
This one has `Optional[bool]` to make it compatible with the `set_ttl` function from the abstract class, right? I'm asking because this implementation itself only returns `None`.